ros_ws/src/car_control/launch/car_control.launch

```diff
@@ -1,42 +1,30 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <launch>
-<!-- 
+  <!-- 
 Launches all necessary nodes that listen for drive_parameters and translate them to the car
 -->
-    <arg name="mode_override" default="0"/>
-    <arg name="dms_check_rate" default="20" /> <!-- in Hz -->
-    <arg name="dms_expiration" default="100" /> <!-- in ms -->
-    <arg name="emergencystop_expiration" default="3000" /> <!-- in ms -->
-	
-    <!-- car control node -->
-    <node
-      respawn="true"
-      pkg="car_control"
-      type="car_controller"
-      name="car_controller"
-      output="screen" >
-    </node>
+  <arg name="mode_override" default="0"/>
+  <arg name="dms_check_rate" default="20" />
+  <!-- in Hz -->
```

Comment:
Why did you put these in separate lines? It makes it harder to see which line they belong to.